tests/test_transforms/test_decomposition/test_base_change_points.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
```

Comment:
Also add tests for `get_change_points_intervals` and `get_change_points` output format